js/listeners.js

```diff
@@ -55,6 +55,34 @@ addEventListener("keydown", (e) => {
           })
         );
       }
+      if (
+    (keys.space.pressed && player.powerUp === "MultiShot" && !game.over)
+    ) {
+    audio.shoot.play();
+    
+    projectiles.push(
+      new Projectile({
```

Comment:
We are going to call `new Projectile` alot, why not add a static function in the Projectile class, 

under class Projectile 

```
  static createMultiShotProjectile(xVelocity) {
    return  new Projectile({
        position: {
          x: player.position.x + player.width / 2,
          y: player.position.y,
        },
        velocity: {
          x: xVelocity,
          y: -10,
        }
      })
```

Reply:
I was having issues with static so I just used function.